samples/daily-prebuilt/permissions-can-admin/src/components/DailyContainer/DailyContainer.js

```diff
@@ -31,16 +31,16 @@ export default function DailyContainer() {
 
   const handleJoinedMeeting = (e) => {
     console.log(e.action);
-    setParticipants((participants) => ({
-      ...participants,
+    setParticipants((p) => ({
+      ...p,
       [e.participants.local.session_id]: e.participants.local,
     }));
   };
 
   const handleParticipantJoined = (e) => {
     console.log(e.action);
-    setParticipants((participants) => ({
-      ...participants,
+    setParticipants((p) => ({
```

Comment:
These (and similar changes below) are related to the eslint error of `participants` already being defined in the outer scope